.data/vSMR_WIP_areas.txt

```diff
@@ -12,25 +12,31 @@ COORD:N052.12.22.287:E000.10.20.976
 COORD:N052.12.28.895:E000.10.28.275
 COORD:N052.12.31.207:E000.10.30.183
 
-EGPH M (Q-S) - NOTAM A2226/25 ends 2025-07-02 2300
-COORD:N055.56.59.105:W003.21.12.808
-COORD:N055.56.54.809:W003.20.58.729
-COORD:N055.56.53.421:W003.20.59.745
-COORD:N055.56.57.790:W003.21.13.948
+EGLL N1 - NOTAM A4416/25 ends 2025-09-30 0430
+COORD:N051.27.57.403:W000.26.02.599
+COORD:N051.27.57.393:W000.26.05.023
+COORD:N051.28.01.250:W000.26.05.068
+COORD:N051.28.01.288:W000.26.02.644
 
-EGLL S (NESSY) - NOTAM A3569/25 ends 2025-07-02
-COORD:N051.27.46.493:W000.26.19.831
-COORD:N051.27.45.049:W000.26.19.462
-COORD:N051.27.45.345:W000.26.11.821
-COORD:N051.27.46.512:W000.26.11.821
+EGLL Link 41 - NOTAM A4416/25 ends 2025-09-30 0430
+COORD:N051.27.44.253:W000.26.11.383
+COORD:N051.27.42.853:W000.26.11.367
+COORD:N051.27.42.840:W000.26.14.599
+COORD:N051.27.44.211:W000.26.14.615
 
-EGLL A(S) (E-F) - AIP SUP 033/2025 ends 2025-10-15
+EGPH G - NOTAM A4416/25 ends 2025-09-30 0700
```

Comment:
I assume this notam name is a typo, NOTAM A4416 refers to EGLL. Think it should be NOTAMR A2226/25.